demo/src/main/java/com/novoda/merlin/demo/presentation/base/MerlinActivity.java

```diff
@@ -50,6 +50,12 @@ protected void onStop() {
         Logger.detach(logHandle);
     }
 
+    @Override
+    protected void onDestroy() {
+        merlin = null;
```

Comment:
Is the idea that we'd have some kind of `Merlin.dispose` method which handles all this, at some point?